<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>General</class>
 <widget class="QWidget" name="General">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos Generales de la Sesion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Legajo N째  Autos  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Acuerdo Reglamentario N째</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="tNueva"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lLegajoN"/>
     </item>
     <item row="5" column="1">
      <widget class="QDateTimeEdit" name="tFecha"/>
     </item>
     <item row="4" column="1">
      <widget class="QTimeEdit" name="tFin"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Nuevo reuni처n, fecha :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lNumeroA"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hora de inicio:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hora de finalizaci처n:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="Guardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Fecha</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="tFechaActual"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lLegajoN</tabstop>
  <tabstop>lNumeroA</tabstop>
  <tabstop>tNueva</tabstop>
  <tabstop>tFin</tabstop>
  <tabstop>tFecha</tabstop>
  <tabstop>Guardar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
